bot/logger/logger.py

```diff
@@ -0,0 +1,60 @@
+"""Logger Module. Configures the logger for the discord library and the logger for custom code.
+It also provides the decorator for logging command calls.
+"""
+import functools
+import logging
+
+import discord
+
+from bot import constants
+
+# configure logger for bot
+log = logging.getLogger("bot")
+log.setLevel(logging.INFO)
+file_handler = logging.FileHandler(filename=constants.LOG_FILE_PATH, encoding='utf-8', mode='w')
+file_handler.setFormatter(logging.Formatter('%(levelname)s %(asctime)s: %(message)s'))
+log.addHandler(file_handler)
+stream_handler = logging.StreamHandler()
+stream_handler.setLevel(logging.WARNING)
+stream_handler.setFormatter(logging.Formatter('%(levelname)s %(asctime)s: %(message)s'))
+log.addHandler(stream_handler)
+
+# add discord to logging (file and console)
```

Comment:
*Add discord.py errors to the logger so that they can be logged as well.

Reply:
Again not a docstring